src/main/java/com/powsybl/openloadflow/FullVoltageInitializer.java

```diff
@@ -32,9 +32,9 @@ public FullVoltageInitializer(VoltageMagnitudeInitializer magnitudeInitializer,
     }
 
     @Override
-    public void prepare(LfNetwork network) {
-        magnitudeInitializer.prepare(network);
-        angleInitializer.prepare(network);
+    public void prepare(LfNetwork network, double lowImpedanceThreshold) {
```

Comment:
You should pass lowImpedanceThreshold through the constructor of VoltageMagnitudeInitializer instead like it is done for the other parameters